paddle/phi/api/yaml/backward.yaml

```
@@ -1019,6 +1019,15 @@
     func : gelu_grad
   composite: gelu_grad(x, out_grad, approximate, x_grad)
 
+- backward_op : global_scatter_grad
+  forward : (Tensor x, Tensor local_count, Tensor global_count, int ring_id=0, bool use_calc_stream=false) -> Tensor(out)
```

```suggestion
  forward : global_scatter (Tensor x, Tensor local_count, Tensor global_count, int ring_id=0, bool use_calc_stream=false) -> Tensor(out)
```
这里应该是目前编译报错的原因，没找到反向对应的前向op